roles/hardening/tasks/sshd.yml

```diff
@@ -0,0 +1,23 @@
+---
+
+- name: Update sshd config file
+  copy:
+    src: sshd_config
+    dest: /etc/ssh/sshd_config
+  become: true
+
+- name: ensure that sshd is enabled and running
```

Comment:
You not going to force a sshd restart if the config changes?

Reply:
Good point